.mailmap

```diff
@@ -1174,6 +1174,7 @@ Prasoon Shukla <prasoon92.iitr@gmail.com>
 Prateek Papriwal <papriwalprateek@gmail.com>
 Praveen Perumal <ppraveen98841@gmail.com> Praveen Perumal <58477724+solvedbiscuit71@users.noreply.github.com>
 Praveen Sahu <povinsahu@gmail.com> povinsahu1909 <povinsahu@gmail.com>
+Prayag <v.prayag2005@gmail.com> vprayag2005 <v.prayag2005@gamil.com>
```

Comment:
Is `Prayag` the name that you want recorded in the AUTHORS file?

Reply:
yes